### DISCUSSION TITLE
Desktop App (Windows, Mac & Linux)

### DISCUSSION THREAD

**Discussion by trymeouteh** [PaulWoitaschek/Voice#1354] [Ideas]
There is not audiobook player app like Voice for the desktop. I do like listening to audiobooks when I am working on the computer and would appreciate an open source audiobook player for the desktop that is simple just like Voice.

This is an idea for anyone out there who is a desktop developer who likes Voice and audiobooks.

**Comment by Najrim** [comment]
This isn't really an issue for Voice, but have you tried [Cozy](https://github.com/geigi/cozy)? I haven't used it much but seems to work well and it's for audio books. It works on Linux, and maybe Mac.

**Comment by mario-tux** [comment]
It would be great to get a common sync mechanism among Voice and Cozy.

**Comment by alexminder** [comment]
> It would be great to get a common sync mechanism among Voice and Cozy.

The https://github.com/izderadicka/audioserve have similar purpose.
Audiobooks can be played on any device and sync position and other metadata.
